messages/context.json

```diff
@@ -41,7 +41,7 @@
   "store/my-users.my-organization.unable-to-delete-title": "Unable to delete organization",
   "store/my-users.my-organization.unable-to-delete-message1": "You have more than one assigned users in organization: ",
   "store/my-users.my-organization.unable-to-delete-message2": "Please delete all the users before deleting this organization",
-  "store/my-users.my-organization.user.already.assigned": "This user is already belongs to some other company",
+  "store/my-users.my-organization.user.already.assigned": "The organization already contains this user.",
```

Comment:
Actually, there is an implicit business rule here: a user cannot be part of two different organizations at the same time. That is the original intention of this message. I agree it needs to be polished. Can you send us another suggestion?

Reply:
This kind of a message It is a privacy error, because the system is giving information from other organizations. This change makes the message generalist

Reply:
An alternative could be: "This user cannot be added".

What do you think?

Reply:
Makes sense, it is a privacy issue for sure. "This user cannot be added" is ok for me.